nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nxserver/config/s3-glacier-blob-provider-config.xml.nxftl

```diff
@@ -6,15 +6,15 @@
             <property name="awsid">${nuxeo.aws.accessKeyId}</property>
             <property name="awssecret">${nuxeo.aws.secretKey}</property>
             <property name="region">${nuxeo.aws.region}</property>
-            <property name="bucket">${nuxeo.glacier.bucket}</property>
-            <property name="bucket_prefix">${nuxeo.glacier.bucket_prefix}/</property>
+            <property name="bucket">${coldstorage.bucket}</property>
+            <property name="bucket_prefix">${coldstorage.bucket_prefix}/</property>
             <property name="directdownload">true</property>
```

Comment:
see above for the naming, as here we are make a AWS S3 integration.

Reply:
could you rename the `s3-glacier-blob-provider-config.xml.nxftl` file to `nuxeo-coldstorage-config.xml.nxftl`

Reply:
could you extract the to a variable [Here](https://github.com/nuxeo/nuxeo-coldstorage/blob/9e06f3752411cc76b69df5e2451c008491e17e45/nuxeo-coldstorage-package/src/main/resources/install/templates/coldstorage/nxserver/config/s3-glacier-blob-provider-config.xml.nxftl#L5)  as a variable `<class>${nuxeo.core.binarymanager}</class>` as we did for Aspera with a default value in the `nuxeo.default`